app/models/user.rb

```diff
@@ -28,17 +36,28 @@ def self.find_create_or_update_by_slack_id!(slack_id)
     instance
   end
 
-  def self.leaderboard(max = 3)
+  def self.standings(max = nil)
     players = User.all.any_of({ :wins.gt => 0 }, :losses.gt => 0)
     players = players.limit(max) if max && max > 0
     players = players.desc(:elo).asc(:_id).to_a
     rank = 1
-    leaderboard = []
+    standings = []
     players.each_with_index do |player, index|
       rank += 1 if index > 0 && players[index - 1].elo != player.elo
-      leaderboard << "#{rank}. #{player}"
+      standings << [rank, player]
     end
-    leaderboard.any? ? leaderboard.join("\n") : 'No players.'
+    standings
+  end
+
+  def self.leaderboard(max = 3)
+    stringify_leaderboard(standings(max))
```

Comment:
If you feel motivated, refactor leaderboard methods into a new `Leaderboard` class, out of `User`.